api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -445,6 +446,39 @@ private void verifyInvitationKey(String invitationKey){
     }
   }
 
+  @Override
+  public ResponseEntity<Void> updateContactEmail(UpdateContactEmailRequest updateContactEmailRequest) {
+    verifyNoLogins(updateContactEmailRequest.getUsername());
+    //resend Welcome Email to new contact email... Add once Mandrill API complete
+    updateUser(updateContactEmailRequest);
+    return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
+  }
+
+  private void verifyNoLogins(String username){
+    User user = userDao.findUserByEmail(username);
+    if (user.getFirstSignInTime() != null) {
+      throw new BadRequestException(
+        "This account has already been in use, if you would like to update your contact email please login and update via the Profile page."
+      );
+    }
+  }
+
+  private void updateUser(UpdateContactEmailRequest updateRequest) {
+    com.google.api.services.admin.directory.model.User googleUser =
+      directoryService.getUser(updateRequest.getUsername());
+    googleUser.setPrimaryEmail(updateRequest.getContactEmail());
+
+    User user = userDao.findUserByEmail(updateRequest.getUsername());
+    user.setContactEmail(updateRequest.getContactEmail());
```

Comment:
Do we want to verify the email address before saving it? In theory, this could bubble back up to the UI letting the user know they tried to update their previous (and potentially invalid) email with another, invalid email.